README.md

```diff
@@ -1,12 +1,72 @@
+
+Thanks for considering helping us! 🤗
+
+If you want to help us in our mission, join our communication channels listed above. 
+
+### How Can I Propose a new Task for SIG Observability?
+
+1. Ensure there is no similar issue already in [GitHub Issues](https://github.com/cncf/sig-observability/issues)
+2. [Add new issue](https://github.com/cncf/sig-observability/issues/new) with explanation.
+3. If the issue is large and you would like to have quicker feedback, add the issue number as a topic for next SIG Observability meeting on our meeting notes.
+
+### How Can I help?
+
+The best way to start is to look [at issues in the backlog on our Kanban Board](https://github.com/cncf/sig-observability/projects/1#card-36042463).
+If you are happy to help in any of those, comment on the issue. Also do not hesitate to ask us on our communication channels or GitHub for more details if needed!
+
+## Operations
+
+### TOC Liaison
+
+| Name                | Email                  |  GitHub                                            | Company                                 |
+|---------------------|------------------------|----------------------------------------------------|-----------------------------------------|
+| Brendan Burns       | `bburns@microsoft.com` | [@brendandburns](https://github.com/brendandburns) | [Microsoft](https://www.microsoft.com/) |                                                                                                          |
+
+### Chairs, in alphabetical order
+
+| Name                | Email                  | CNCF Slack         | GitHub                                         | Company                                 | Open Source                                                                                              |
+|---------------------|------------------------|--------------------|------------------------------------------------|-----------------------------------------|----------------------------------------------------------------------------------------------------------|
+| Matt Young          | `myoung@everquote.com` | `@Matt Young`      | [@halcyondude](https://github.com/halcyondude) | [Everquote](https://www.everquote.com/) |                                                                                                          |
+| Richard Hartmann    | `richardhartmann.de`   | `@RichiH`          | [@RichiH](https://github.com/RichiH)           | [Grafana](https://grafana.com/)         |  Prometheus [Team](https://prometheus.io/governance/#team-members); [PromCon](https://promcon.io/) Lead  |
+
+### Tech Lead
+
+| Name                | Email                  | CNCF Slack         | GitHub                                         | Company                                 | Open Source              |
+|---------------------|------------------------|--------------------|------------------------------------------------|-----------------------------------------|--------------------------|
```

Comment:
I don't think tables add much value here and would prefer to avoid them, but I don't feel strongly either way.

Reply:
I really like them. Those really give very clean look and are very readable. The key is also to have contact details, (even though all of you have such detail in your GH profile).

I would vote to have that :hugs:

Reply:
Also don't see harm. (: